sql/catalyst/src/test/scala/org/apache/spark/sql/catalyst/expressions/CollectionExpressionsSuite.scala

```diff
@@ -1854,4 +1854,18 @@ class CollectionExpressionsSuite extends SparkFunSuite with ExpressionEvalHelper
       Literal(stringToInterval("interval 1 year"))),
       Seq(Date.valueOf("2018-01-01")))
   }
+
+  test("SPARK-32133: Sequence step must be a day interval " +
```

Comment:
instead of adding a new test, shall we just put the new negative test in the existing test case `Sequence of dates`? We can point to the JIRA number in the code comment.

Reply:
Thanks, I have moved the negative test to `Sequence of dates` and ` point to the JIRA number in the code comment.`